src/main/java/helper/GoogleUserMapper.java

```diff
@@ -0,0 +1,33 @@
+package helper;
```

Comment:
Instead of just having the Google Auth classes in the package `helper`, let's have them in their own package/sub-package?
WDYT @sankalpbansal-google ?

Reply:
Made a new package named `googlesignin`.